addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -30,26 +30,14 @@ var SmsWidget = FieldTextEmojis.extend({
         if (this.enableEmojis) {
             this._super.apply(this, arguments);
         }
-    },
-
-    //--------------------------------------------------------------------------
-    // Private: override widget
-    //--------------------------------------------------------------------------
-
-    /**
-     * @private
-     * @override
-     */
-    _renderEdit: function () {
-        var def = this._super.apply(this, arguments);
-
-        this._compute();
-        var $sms_container = $('<div class="o_sms_container"/>');
-        $sms_container.append(this._renderSMSInfo());
-        $sms_container.append(this._renderIAPButton());
-        this.$el = this.$el.add($sms_container);
-
-        return def;
+        if (this.mode === 'edit') {
```

Comment:
Do we need all those changes just to remove the extra underline?
Seems unlikely, looks like you just re-used your previous code.
(The commit message even mentions the focus issue while it should be fixed already).

Please investigate only the underline issue and make a fix with the minimum amount of code changes.